eo/notebook_config.yml

```diff
@@ -0,0 +1,10 @@
+# Config file containing the list of notebooks to download for a specific image.
+#
+# Used in the deploy-data-specific-image script from pavics-jupyter-base.
+# More details on this config can be found on this script file :
+#   https://github.com/bird-house/pavics-jupyter-base/blob/master/scheduler-jobs/deploy_data_specific_image
+
+- repo_url: https://github.com/crim-ca/pavics-jupyter-images
+  branch: master
+  source_path: eo/notebooks
+  dest_sub_dir:
```

Comment:
I would expect `dest_sub_dir: crim-eo`?  But I am not sure.

Reply:
@tlvu see my comment here : https://github.com/bird-house/pavics-jupyter-base/pull/3#discussion_r625983632
Like this, the notebooks would be checkout to : /data/jupyterhub_user_data/tutorial-notebooks/crim-eo
What I asked to @ChaamC would make it clearer to show how to achieve the following strucutre: 
/data/jupyterhub_user_data/tutorial-notebooks/crim-eo/eo
/data/jupyterhub_user_data/tutorial-notebooks/crim-eo/common

Reply:
I can add the suggested structure. 
But what would be the repo_url/branch/source_path for the common folder?
Should I try to find all the notebooks that are found in the tutorial-notebooks folder (when spawning the pavics-workflow image for example)?
Or maybe I can just add this [folder](https://github.com/Ouranosinc/pavics-sdi/tree/master/docs/source/notebooks) for now?

Reply:
I thought that the current tutorial-notebooks folder was filled using only that folder. But looking at the diff, this is enough for our common folder. Maybe we could call it "core"?

Reply:
Yes, there are more in the current tutorial-notebooks folder, so more sources are necessary if we want them all. But I don't think all those notebooks will work anyway. The eo image for example has only the base packages and the specific packages for eo. The packages necessary  for the current tutorial-notebooks folder are not all on the eo image.

Even with the linked folder in my previous comment, I am not sure which notebooks will work or not from that list.

So, do I just take the linked folder for now, and store it in a 'core' folder, for both eo/nlp?

Reply:
Yes go for it. At least for people using those images, they will see what PAVICS has to offer and the mecanism will be demonstrated. If some libraries are missing we will fix that later.

Reply:
FYI info if you guys want to replicate exactly what in the current `tutorial-notebooks` folder into your `common` folder https://github.com/bird-house/birdhouse-deploy/blob/20d440fd293b01d4cf3e1dccf2933877a651d571/birdhouse/deployment/trigger-deploy-notebook#L60-L71

However, keep in mind that bringing in notebook that your image can not run (missing packages) is no cool !   I'd suggest keeping only the notebooks relevant to your image runtime capabilities.

With that in mind, pavics-sdi is a good start (probably most notebooks will run without missing packages).  Finch and Raven notebooks will requires more packages to be installed.

Reply:
> However, keep in mind that bringing in notebook that your image can not run (missing packages) is no cool !

I agree, but it's more as a place holder or demonstration capabilities right now. I think that once this setup is ready we will be able to better partitionize the notebooks for all our (Ouranos and CRIM) images.

Reply:
Common folder has been added to configs :)